javascript/javascript2/week3/weatherApp/style1.css

```diff
@@ -0,0 +1,80 @@
+body {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0;
+    background: #222;
+    background-image: url(https://source.unsplash.com/1600x900/?landscape);
```

Comment:
There is one css style that deals with not repeating the background image as here. Try that.